flow/core/kernel/vehicle/traci.py

```diff
@@ -18,6 +18,11 @@
 WHITE = (255, 255, 255)
 CYAN = (0, 255, 255)
 RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+STEPS = 10
+rdelta = 255 / STEPS
+# smoothly go from red to green as the speed increases
+color_bins = [[int(255 - rdelta * i), int(rdelta * i), 0] for i in range(STEPS + 1)]
```

Comment:
this is cool, but SUMO does this by default btw :^)

Reply:
How do I turn this on in SUMO?

Reply:
Can you replace this with the correct command?

Reply:
non-trivial to implement, have to add a gui-settings file and manually set that as the default setting for vehicle coloring

i.e. https://sumo.dlr.de/docs/SUMO-GUI.html#configuration_files